Add space to extent_test_chars to have FontAwesome being recognized.

Fixes:

    $ urxvt -fn "xft:FontAwesome"
    urxvt: unable to calculate font width for 'FontAwesome:minspace=True', ignoring.
    urxvt: unable to load base fontset, please specify a valid one using -fn, aborting.

Posted to upstream mailinglist:

http://lists.schmorp.de/pipermail/rxvt-unicode/2016q4/002308.html
Followed up in http://lists.schmorp.de/pipermail/rxvt-unicode/2017q4/002441.html

 src/rxvtfont.C | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rxvtfont.C b/src/rxvtfont.C
index c56921c..5932f03 100644
--- a/src/rxvtfont.C
+++ b/src/rxvtfont.C
@@ -155,7 +155,7 @@ static const struct rxvt_fallback_font {
 // these characters are used to guess the font height and width
 // pango uses a similar algorithm and doesn't trust the font either.
 static uint16_t extent_test_chars[] = {
-  '0', '1', '8', 'a', 'd', 'x', 'm', 'y', 'g', 'W', 'X', '\'', '_',
+  ' ', '0', '1', '8', 'a', 'd', 'x', 'm', 'y', 'g', 'W', 'X', '\'', '_',
   0x00cd, 0x00d5, 0x0114, 0x0177, 0x0643,	// ÍÕĔŷﻙ
   0x304c, 0x672c,				// が本
 };
-- 
2.10.0

